assets/js/base/components/cart-checkout/product-metadata/index.js

```diff
@@ -22,9 +23,12 @@ const ProductMetadata = ( {
 				shortDescription={ shortDescription }
 				fullDescription={ fullDescription }
 			/>
-			<ProductVariationData
-				className="wc-block-components-product-metadata__variation-data"
-				variation={ variation }
+			<ProductDetails details={ itemData } />
```

Comment:
Why do we need two `<ProductDetails>`?

Reply:
According to the designs there should be some margin between the item data added by extensions and variations, so I thought an easy (and semantic) way of doing that was to render them in two different lists. But I don't have a strong opinion on this, alternatively we could render all in the same list and then add the margin with CSS.

Reply:
OK if it is design-driven.